components/Banner/src/MDCBannerView.m

```diff
@@ -271,6 +272,8 @@ - (void)setUpButtonsConstraints {
       constraintGreaterThanOrEqualToAnchor:self.buttonContainerView.leadingAnchor];
   self.leadingButtonConstraintTop =
       [self.leadingButton.topAnchor constraintEqualToAnchor:self.buttonContainerView.topAnchor];
+  self.leadingButtonConstraintBottom = [self.leadingButton.bottomAnchor
```

Comment:
Tip: Can you use `firstBaselineAnchor` or `lastBaselineAnchor` anchors instead to ensure the text is aligned?  I'm concerned that two different button styles may result in different text alignment if only the bottoms are aligned.

Reply:
I think it is a good idea! while I just doubled checked the spec, it requires two buttons to be 8pt above the bottom and required two button to be the same style. I think it is safer to align them by the bottom anchor according to the spec for now?

Reply:
I would still recommend baseline alignment, unless you prefer the text to be unaligned. Since the button could present two languages (e.g., a translation option), the text may render differently within the buttons.

Reply:
Talked offline. Opened an [issue](https://github.com/material-components/material-components-ios/issues/8326) to track. Keeping the buttons aligned with bottom meets the design guidelines at the current stage.

Reply:
Ack.